src/main/java/Sheep.java

```diff
@@ -0,0 +1,13 @@
+public class Sheep implements Domesticatable, Tradable{
+    /* A Sheep class that implements the interface Domesticable and Drivable
```

Comment:
Maybe specify a constructor after this comment here.

Reply:
The class has implemented Tradable not Drivable. The documentation need to be changed.